pandas/io/formats/style.py

```diff
@@ -3166,10 +3165,10 @@ def text_gradient(
         return self.apply(
             _background_gradient,
             cmap=cmap,
-            subset=subset,
-            axis=axis,
             low=low,
             high=high,
+            axis=axis,
```

Comment:
No real reason, I believe I was trying to match the order of the parameters in the docstring.

Reply:
Mind reverting this change for now (could be a follow up PR)? We want to avoid any potential unrelated behavior change as this was supposed to just adjust a docstring.

Reply:
Sure, will do

Reply:
I think this is a valid change, but can be easier to add it in a separate taregted PR after this one.

Reply:
reverted the unnecessary changes